contrib/devtools/symbol-check.py

```diff
@@ -16,30 +16,30 @@
 import sys
 import os
 
-# Debian 6.0.9 (Squeeze) has:
+# Debian 8 (Jessie) has:
 #
-# - g++ version 4.4.5 (https://packages.debian.org/search?suite=default&section=all&arch=any&searchon=names&keywords=g%2B%2B)
-# - libc version 2.11.3 (https://packages.debian.org/search?suite=default&section=all&arch=any&searchon=names&keywords=libc6)
-# - libstdc++ version 4.4.5 (https://packages.debian.org/search?suite=default&section=all&arch=any&searchon=names&keywords=libstdc%2B%2B6)
+# - g++ version 4.9.2 (https://packages.debian.org/search?suite=jessie&arch=any&searchon=names&keywords=g%2B%2B)
+# - libc version 2.19 (https://packages.debian.org/search?suite=jessie&arch=any&searchon=names&keywords=libc6)
 #
-# Ubuntu 10.04.4 (Lucid Lynx) has:
+# Ubuntu 16.04 (Xenial) has:
```

Comment:
```suggestion
# Ubuntu 16.04 Xenial. EOL: April, 2021. https://help.ubuntu.com/community/EOL#Ubuntu_16.04_Xenial_Xerus
```

Reply:
@MarcoFalke there seems to be some ambiguity about the EOL date for 16.04, this page says it's April 2024, and April 2021 is only the "End of Standard Support":

https://wiki.ubuntu.com/Releases

Reply:
I've changed it to use the 2024 date.